### DISCUSSION TITLE
Mushroom drop rates Windows Vs Linux

### DISCUSSION THREAD

**Discussion by Akinraze** [flareteam/flare-game#873] [Q&A] [answered]
Hello

I have recently installed this game on both Windows 10 and Linux Kubuntu 20.04.3 LTS. Unsure if they are the same version. Install for windows came from https://flarerpg.org/download/, Linux version was installed via Discover within Linux.

Windows has a very good drop rate of Mushrooms. I have a level 3 player which has farmed 60+ mushrooms in one day. The player has reached level 3, while on Linux I have zero drop rate all the way up to level 12 player. While I do not mind farming hard to get a few Mushrooms on Linux I find it disappointing that I have found zero. Is there anyway I can boost the drop rate on Linux?

**Comment by dorkster** [comment, marked as answer]
There is no difference in drop rates between platforms. Mushrooms were introduced in version 1.12, so make sure that's the version of the game you're playing. If you installed the Flatpak package via Discover, it should be 1.12. It took about 6 minutes for me to get my first drop of Mushrooms with that version of the game.

You might have just been very unlucky. The drop rates for crafting items range between 10-16%. The probability of specifically getting Mushrooms is lower, because the drop rate is shared with the other crafting items (and technically Gold, which has a higher drop rate of 50%). So at a minimum, if you have a chance to get Mushrooms, it's split 4-ways between Gold, Aloe Vera, Mana Crystal, and Mushrooms.

**Reply by Akinraze** [reply]
Thank you Dorkster. I found a more up to date version and installed it.
